src/encord_active/lib/db/prisma_init.py

```diff
@@ -0,0 +1,28 @@
+import filecmp
+from pathlib import Path
+
+from prisma import __file__ as prisma_module_file
+from prisma.cli.prisma import run
+
+from encord_active.lib.common.decorators import silence_stdout
+
+prisma_run = silence_stdout(run)
+
+GENERATED_PRISMA_SCHEMA_FILE = Path(prisma_module_file).parent / "schema.prisma"
+PRISMA_SCHEMA_FILE = Path(__file__).parent / "schema.prisma"
+
+
+def ensure_prisma_db(db_path: Path):
+    env = {"MY_DATABASE_URL": f"file:{db_path}"}
+    failure = prisma_run(["migrate", "deploy", f"--schema={PRISMA_SCHEMA_FILE}"], env=env)
+    # TODO: remove me before release
```

Comment:
what about this todo? is ok to go with it to main?

Reply:
we want it to go to main for a short while so everyone who already used prisma can start all projects and reset them. after that we can remove